config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -73,6 +72,15 @@ function create_libvirt_image() {
 # Function to dowload the rhel base image
 
 function download_rhel_kvm_guest_qcow2() {
+    #Validate RHEL version for IBM Z Secure Enablement
+    if [ "$SE_BOOT" == "true" ]; then
+        version=$(echo $BASE_OS_VERSION | awk -F "." '{ print $1 }')
+        release=$(echo $BASE_OS_VERSION | awk -F "." '{ print $2 }')
+        if [[ "$version" -lt 9 || ("$version" -eq 9 && "$release" -lt 4) ]]; then
+            error_exit "Libvirt Secure Execution supports RHEL OS version 9.4 or above"
```

Comment:
@ANJANA-A-R-K  if min base os version for SE is 9.4, shouldn't the default be changed accordingly here
https://github.com/openshift/sandboxed-containers-operator/blob/ff025162858eb5040d26cfda1e57520580983876/config/peerpods/podvm/libvirt-podvm-image-cm.yaml#L24

Reply:
I've changed it to 9.4 in libvirt-podvm-image-cm.yaml